services/net/notification/NotificationManager.cs

```diff
@@ -93,8 +97,7 @@ public class NotificationManager : ServiceManager<NotificationOptions>
         this.Listener.IsLongRunningJob = true;
         this.Listener.OnError += ListenerErrorHandler;
         this.Listener.OnStop += ListenerStopHandler;
-        _engine = engine;
-        _notificationOptions = notificationOptions;
```

Comment:
@CelloLiu Options are already part of the base class.  You don't need to duplicate them.  Additionally, you should extract the `notificationOptions.Value` in the constructor based on examples from Microsoft.